src/components/select/select.scss

```diff
@@ -3,9 +3,6 @@
 @use '../../style/mixins';
 
 @use '@material/select/styles';
-@use '@material/select/helper-text/mdc-select-helper-text';
-@use '@material/floating-label';
-@use '@material/floating-label/mdc-floating-label';
```

Comment:
First of all, MDC wants us to use another set of styles:
https://github.com/material-components/material-components-web/tree/69fd619d0cf28dc6588650282bc825858b104a74/packages/mdc-select
which are:
```scss
@use "@material/list/mdc-list";
@use "@material/menu-surface/mdc-menu-surface";
@use "@material/menu/mdc-menu";
@use "@material/select/styles";
```
But neither those we used to have, nor the ones MDC suggests
have any effect. I guess this is due to us having a custom implementation
of their `select` component. So I decided to remove them.